src/Mobs/Spider.cpp

```diff
@@ -43,9 +43,15 @@ void cSpider::EventSeePlayer(cEntity * a_Entity)
 		return;
```

Comment:
It checks when it reaches the head of the queue, if the chunk needs lighting, so there is a risk a chunk will be kept loaded for longer then necessary.